source/drivers/php.txt

```diff
@@ -16,69 +16,29 @@ To access MongoDB from PHP you need:
 
 - The MongoDB PHP driver installed.
 
-Install the PHP Driver
-----------------------
+Drivers
+-------
 
-\*NIX
-~~~~~
+The currently maintained driver is the **mongodb** extension available from
+PECL. This driver can be used stand-alone, although it is very bare-bones. You
+should however use the driver through the complimentary *PHP library*, which
+implements a more convenient API on top of the bare-bones driver.
```

Comment:
"You should consider using the driver with the complimentary *PHP library*, which implements a more full-featured API atop the bare-bones driver."